configuration/scripts/tests/base_suite.ts

```diff
@@ -4,6 +4,7 @@ smoke          gx3     1x1        debug,diag1,run2day
 smoke          gx3     1x4        debug,diag1,run2day
 smoke          gx3     4x1        debug,diag1,run5day
 restart        gx3     8x2        debug
+smoke          gx1    64x1        qc,medium
```

Comment:
I'm not sure about this. The longest test currently in base_suite is `smoke          gx3     8x2        diag24,run1year,medium` which runs 1 year. This adds the QC test which runs for 5 years, which is considerably more..

If the base suite is bit4bit, we do not need to run a 5 year test, no ? If the base_suite finds non-bit4bit changes then it makes sense to run the QC  test... I think I would prefer keeping this separate.